Cargo.toml

```diff
@@ -39,4 +38,5 @@ features = [
 ]
 
 [dev-dependencies]
-rand = "0.4"
+rand = "0.8.0"
+socket2 = "0.3.18"
```

Comment:
Have you tried compiling this? For me it fails because of the `extern crate socket2` in `lib.rs`.